docs/tutorials/using-ps3-sixaxis.md

```diff
@@ -0,0 +1,103 @@
+
+
+# Installing the necessary python libraries
+To read gamepad values into our python program we need to install the evdev library. (This is not a typo. There is no 3 in evdev)
+
+Install it like so:
+    sudo apt-get update
+    sudo apt-get install python-dev python-pip gcc
+    sudo apt-get install linux-headers-$(uname -r)
+    sudo pip install evdev
+
+The last command will take a long time and won't show much response. Be patient!
+
+# Running motors with a PS3 sixaxis controller
+Here's a quick program that will take the right stick Y axis and use it to set the speed of a motor in port A. Note that motor control is in a separate thread. That's because controlling the motors is much slower than reading the gamepad. Multithreading synchronizes both.
+
+    __author__ = 'Anton Vanhoucke'
+    
+    import evdev
+    import ev3dev.auto as ev3
+    import threading
+    
+    ## Some helpers ##
+    def scale(val, src, dst):
+        """
+        Scale the given value from the scale of src to the scale of dst.
+    
+        val: float or int
+        src: tuple
+        dst: tuple
+    
+        example: print scale(99, (0.0, 99.0), (-1.0, +1.0))
+        """
+        return (float(val - src[0]) / (src[1] - src[0])) * (dst[1] - dst[0]) + dst[0]
+    
+    def scale_stick(value):
+        return scale(value,(0,255),(-100,100))
+    
+    ## Initializing ##
+    print "Finding ps3 controller..."
+    devices = [evdev.InputDevice(fn) for fn in evdev.list_devices()]
+    for device in devices:
+        if device.name == 'PLAYSTATION(R)3 Controller':
+            ps3dev = device.fn
+    
+    gamepad = evdev.InputDevice(ps3dev)
+    
+    speed = 0
+    running = True
+    
+    class MotorThread(threading.Thread):
+        def __init__(self):
+            self.motor = ev3.LargeMotor(ev3.OUTPUT_A)
+            threading.Thread.__init__(self)
+    
+        def run(self):
+            print "Engine running!"
+            while running:
+                self.motor.run_forever(duty_cycle_sp=speed)
+    
+            self.motor.stop()
+    
+    motor_thread = MotorThread()
+    motor_thread.setDaemon(True)
+    motor_thread.start()
+    
+    
+    for event in gamepad.read_loop():   #this loops infinitely
+        if event.type == 3:             #A stick is moved
+            if event.code == 5:         #Y axis on right stick
+                speed = scale_stick(event.value)
+    
+        if event.type == 1 and event.code == 302 and event.value == 1:
+            print "X button is pressed. Stopping."
+            running = False
+            break
+
+
+# The complete event type and code mapping of the ps3 controller
+I mapped out all codes for you! Here they are:
+{% include screenshot.html source="/images/Website/sixaxis_event_codes.png" %}
+
+# The result: a remote controle robot. 
+How cool! No computer needed. Just a gamepad and the ev3 brick. Watch it on youtube: https://www.youtube.com/watch?v=brfgF3D5c4k
```

Comment:
If you want to embed the video, you can write this instead of the url

    {% include youtube-embed.html youtube_video_id="brfgF3D5c4k" %}

Other wise you need to write the url as 

    <https://www.youtube.com/watch?v=brfgF3D5c4k>

or 

    [Watch it on youtube](https://www.youtube.com/watch?v=brfgF3D5c4k)